tests/foreman/cli/test_satellitesync.py

```diff
@@ -1669,7 +1669,6 @@ def test_positive_export_rerun_failed_import(
         assert len(importing_cvv) == 1
 
     @pytest.mark.tier3
-    @pytest.mark.skip_if_open("BZ:2262379")
     def test_postive_export_import_ansible_collection_repo(
```

Comment:
```suggestion
    @pytest.mark.skip_if_open("SAT-23051")
    def test_postive_export_import_ansible_collection_repo(
```

Reply:
Since this issue is open, should we still use the pytest marker with the SAT issue instead?
Should work the same as the BZ being passed.

Reply:
Seems they function the same, non-blocking